src/shared/settings.js

```diff
@@ -33,8 +33,16 @@ function jsonConfigsFrom(document) {
 
   var config = {};
   for (var i=0; i < settingsElements.length; i++) {
-    assign(config, JSON.parse(settingsElements[i].textContent));
+    var settings;
+    try {
+      settings = JSON.parse(settingsElements[i].textContent);
+    } catch (err) {
+      console.warn('Could not parse settings from js-hypothesis-config tags', err);
+      settings = {};
+    }
+    assign(config, settings);
```

Comment:
Log a warning for each unparseable js-hypothesis-config, but still read other valid ones